crt_portal/cts_forms/tests/test_forms.py

```diff
@@ -24,6 +30,54 @@ def test_valid(self):
         })
         self.assertTrue(form.is_valid())
 
+    def test_user_assignment(self):
+        form = ComplaintActions(
+            initial={
+                'assigned_section': 'ADM',
+                'status': 'new',
+                'primary_statute': '144',
+                'district': '1',
+                'assigned_to': self.user1.pk
+            },
+            data={
+                'assigned_section': 'ADM',
+                'status': 'new',
+                'primary_statute': '144',
+                'district': '1',
+                'assigned_to': self.user2.pk
+            }
+        )
+
+        self.assertTrue(form.is_valid())
+
+        for action in form.get_actions():
+            self.assertEqual(action[0], 'Assigned to:')
+            self.assertEqual(action[1], f'Updated from "{self.user1.username}" to "{self.user2.username}"')
+
+    def test_user_new_assignment(self):
+        form = ComplaintActions(
+            initial={
+                'assigned_section': 'ADM',
+                'status': 'new',
+                'primary_statute': '144',
+                'district': '1',
+                'assigned_to': None,
+            },
+            data={
+                'assigned_section': 'ADM',
+                'status': 'new',
+                'primary_statute': '144',
+                'district': '1',
+                'assigned_to': self.user2.pk,
+            }
+        )
+
+        self.assertTrue(form.is_valid())
+
+        # Running the code to check error. No output to test.
+        for action in form.get_actions():
+            pass
```

Comment:
@ilodidoj - we should be able to test some output here. Much like `test_user_assignment` above, there should be a single "Assigned to" action coming back from `get_actions`. To get a feel for how to write the test, I'd suggest simply printing the action to the console here, but in the end it should look pretty similar to lines 54-55 above. 

Let me know if you want to pair on this, would be happy to!